tests/forms_tests/tests/test_forms.py

```diff
@@ -3866,3 +3799,154 @@ class CustomForm(Form):
         custom = CustomRenderer()
         form = CustomForm(renderer=custom)
         self.assertEqual(form.renderer, custom)
+
+
+# RemovedInDjango50
+class DeprecationTests(SimpleTestCase):
+    def test_depreation_warning(self):
+        msg = 'django.forms.BaseForm._html_output` is deprecated. Please use the `render` and `get_context` methods.'
+        with self.assertRaisesMessage(RemovedInDjango50Warning, msg):
+            form = Person()
+            form._html_output(
+                normal_row='<p id="p_%(field_name)s"></p>',
+                error_row='%s',
+                row_ender='</p>',
+                help_text_html=' %s',
+                errors_on_separate_row=True,
+            )
+
+
+# RemovedInDjango50
+class DeprecatedTests(SimpleTestCase):
+    @ignore_warnings(category=RemovedInDjango50Warning)
```

Comment:
Can this decorator be placed on the entire class rather than repeating for each test method?